setup.py

```diff
@@ -444,7 +444,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'parameterized',
     'paramiko',
     'pre-commit',
-    'pylint~=2.4',
+    'pylint==2.4.4',
```

Comment:
```suggestion
    'pylint~=2.4,<2.5.0',
```

Reply:
pylint~=2.4 -> `pylint>=2.4,==2.*` so you add <2.5.0 with it :)

Reply:
Ah okay - get it 👍